designs/administrator/4.5/quay-image-vulnerabilities/index.md

```diff
@@ -0,0 +1,22 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Quay Container Security Operator Enhancements
+
+Enhancements have been made to the Container Security Operator, with the **Image Vulnerabilities** status on the cluster dashboard now having clearer labelling and access to the **Image Vulnerabilities** list. Also, Pod Details now include a **Vulnerabilities** tab which lists out the vulnerable containers on the pod.
+
+## Image Vulnerabilities on Cluster Dashboard
+
+![Cluster Dashboard Image Vulnerabilities Status](img/imagevulndash45.png)
```

Comment:
Small detail - but just a few icon questions:

The icon for the high vulnerability in the popover is different than the one on the pods page. Is that intentional? 

Is there a difference in meaning between the red circle icon next image vulnerabilities in the status card?

Reply:
Good catch, fixed the pod page icon.

I talked with Andy and he agreed that there's no reason the status card icon couldn't match what the popover is using. Will definitely keep this [in mind for 4.6.](https://docs.google.com/document/d/1yrzPFnH6KeCqfwNt3g_B8UsjmYr0MTCn4jQydIahmU0/edit?usp=sharing) Thanks.